remoteappmanager/file_config.py

```diff
@@ -94,4 +132,35 @@ def parse_config(self, config_file):
                 'Could not find specified configuration'
                 ' file "{}"'.format(config_file))
 
-        set_traits_from_dict(self, file_line_parser)
+        set_traits_from_dict(self, file_line_parser.as_dict())
+
+        if self.tls or self.tls_verify:
+            self.docker_host = self.docker_host.replace('tcp://', 'https://')
+
+    def docker_config(self):
+        """Extracts the docker configuration as a dictionary suitable
+        to be passed as keywords to the docker client.
+        """
+        params = {}
+        params["base_url"] = self.docker_host
+
+        # Note that this will throw if the certificates are not
+        # present at the specified paths.
+        # Note that the tls flag takes precedence against tls verify.
+        # This is docker behavior.
+        if self.tls:
+            params["tls"] = tls.TLSConfig(
+                client_cert=(self.tls_cert, self.tls_key),
```

Comment:
ok, but we never have tls=True. we have it in the file config but our meaning is the same as the —tls and —tlsverify in docker as a result, we always pass a TLSConfig to the client or nothing, in that case it’s False